straxen/analyses/records_matrix.py

```diff
@@ -85,7 +85,7 @@ def raw_records_matrix(context, run_id, raw_records, time_range,
     # at the start of the range due to missing zeroth fragments
     records = strax.raw_to_records(raw_records)
     strax.baseline(records, allow_sloppy_chunking=True)
-    strax.zero_outside_bounds(records)
+    strax.zero_out_of_bounds(records)
```

Comment:
This was the reason to reinstate more checks. Why was there a function here for over half a year that was bugged

Reply:
Now you also know how many people used this function.

Reply:
haha, problem is that I ran into it so it's worth fixing right?

Reply:
:D Yes no arguing with that.